.github/ISSUE_TEMPLATE/Feature_Request_Windows_Customization.yml

```diff
@@ -0,0 +1,71 @@
+name: "Suggest a new setting/feature within Windows Customization"
+description: Suggest a new feature or improvement within Windows Customization (this does not mean you have to implement
+  it)
+labels:
+- Issue-Feature
+- Area-Windows-Customization
+body:
+- id: minwindowsversion
+  type: input
+  attributes:
+    label: Minimum Supported Windows Version
+    placeholder: "10.0.23400.0"
+    description: What is the minimum version of Windows that should support this feature?
+  validations:
+    required: true
+
+- id: maxwindowsversion 
```

Comment:
Why would someone want to enter a max version?  I would want the change to always work on every version in the future.

Reply:
If there is a known maximum version where the change no longer is applicable or no longer works. We may have to work within these constraints.